tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -286,6 +291,7 @@ def test_h_user_raises_if_user_id_is_missing(self, pyramid_request):
         with pytest.raises(
             HTTPBadRequest, match='Required parameter "user_id" missing from LTI params'
         ):
+            # pylint:disable=expression-not-assigned
```

Comment:
Is the approach here to always do it at the line level? In other tests I think we've put very frequently occurring ones at the top of the file.

Reply:
I tend to be as minimal as possible about the scope of the ignore, though sometimes it's sensible to make an exception to that.

I was too lazy to make the real change here which is that these properties that may raise should probably be methods, not properties. I think pylint may be indirectly pointing out to us that it's not good for properties to raise.